src/auth/auth.proto

```diff
@@ -218,8 +218,17 @@ enum Permission {
   REPO_ADD_PIPELINE_READER    = 212;
   REPO_REMOVE_PIPELINE_READER = 213;
   REPO_ADD_PIPELINE_WRITER    = 214;
+  REPO_CREATE = 215;
 
   PIPELINE_LIST_JOB     = 301;
+  PIPELINE_CREATE = 302;
```

Comment:
these pipeline permissions shouldn't be necessary. We use REPO perms to imply them